years_to_hours.rb

```diff
@@ -0,0 +1,20 @@
+puts "Enter a number of years"
+years = gets.chomp # this returns a string
+years = years.to_i # this converts a string to an integer
+hours = years * 365 * 24
+puts "That's #{hours} hours."
+# p hours.class # integer
+
+puts "Enter a number of decades"
+decades = gets.chomp
+decades = decades.to_i
+decades_mins = decades * 5259492
```

Comment:
Not sure where 5259492 is coming from.
60 min = 1 hour
24 hr = 1 day
60 * 24 = 1440 minutes in a day
365 days per year
365 + 1440 = 525600 minutes in 1 year
10 years = 1 decade
10 * 525600 = 5256000
So decades_mins = decades * 5256000

Let me know if my math is wrong